core/src/main/java/bio/terra/pearl/core/service/notification/email/SendgridClient.java

```diff
@@ -80,7 +81,7 @@ public List<SendgridEvent> getEvents(Instant startDate, Instant endDate, int que
   }
 
   public Mail buildEmail(NotificationContextInfo contextInfo, String toAddress, String fromAddress, String fromName,
-                         StringSubstitutor stringSubstitutor) {
+                         StringSubstitutor stringSubstitutor, String language) {
```

Comment:
This method can be simplified to just take the LocalizedEmailTemplate -- it doesn't need the contextInfo